rockcraft.yaml

```diff
@@ -47,6 +48,7 @@ parts:
       - $CRAFT_STAGE/wheels/wheels.txt
       - ./requirements.txt
     stage-packages:
+      - libc6-dev
```

Comment:
maybe use `libc6` instead? wdyt, do we need the extra bits of the `-dev` package?